src/sentry/api/endpoints/release_file_details.py

```diff
@@ -70,6 +72,13 @@ class ReleaseFileDetailsEndpoint(ProjectEndpoint):
     doc_section = DocSection.RELEASES
     permission_classes = (ProjectReleasePermission,)
 
+    def download(self, releasefile):
+        file = releasefile.file.getfile()
+        response = HttpResponse(FileWrapper(file), mimetype='application/octet-stream')
```

Comment:
We should do the same thing that Django does here for static file delivery, since it's the same idea: https://github.com/django/django/blob/1.6.11/django/views/static.py#L67-L68

The reason for this is if it's this `CompatibleStreamingHttpResponse`, we can actually stream the bytes back to the browser instead of blocking process while it downloads in memory. This streaming behavior is then also handled directly by uWSGI and deferred away from Django.

This happens here: https://github.com/getsentry/sentry/blob/master/src/sentry/wsgi.py#L44